README.md

```diff
@@ -7,25 +7,18 @@ Just some Scala libraries that started out as internal projects as part of the [
 
 ## Download
 
-sphere-json: [ ![Download](https://api.bintray.com/packages/commercetools/maven/sphere-json/images/download.svg) ](https://bintray.com/commercetools/maven/sphere-json/_latestVersion)
+sphere-json: [![latest release](https://img.shields.io/maven-central/v/com.commercetools/sphere-json_2.13.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:com.commercetools%20AND%20a:sphere-json*)
 
-sphere-util: [ ![Download](https://api.bintray.com/packages/commercetools/maven/sphere-util/images/download.svg) ](https://bintray.com/commercetools/maven/sphere-util/_latestVersion)
 
-sphere-mongo: [ ![Download](https://api.bintray.com/packages/commercetools/maven/sphere-mongo/images/download.svg) ](https://bintray.com/commercetools/maven/sphere-mongo/_latestVersion)
+sphere-util: [![latest release](https://img.shields.io/maven-central/v/com.commercetools/sphere-util_2.13.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:com.commercetools%20AND%20a:sphere-util*)
 
-## Documentation
 
-[sphere-json](json/README.md)
+sphere-mongo: [![latest release](https://img.shields.io/maven-central/v/com.commercetools/sphere-mongo_2.13.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:com.commercetools%20AND%20a:sphere-mongo*)
 
-## Including in the build
 
-The library is published to the [Bintray](https://bintray.com/commercetools/maven):
+## Documentation
 
-    resolvers += Resolver.bintrayRepo("commercetools", "maven")
```

Comment:
Removed since addtional resolvers are not required with maven central.

The `sbt` snippet below was removed assuming the badges are good enough, the maven central page has `sbt` on the artifiact / version page. Based on that I removed `project/UpdateVersionInFiles.scala`